.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,79 @@
+# .github/workflows/goreleaser.yml
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, windows-2019, windows-2022]
```

Comment:
Dunno, if we need to think about it, but what happens if one OS fails to push the image? We clean it up afterwards by hand?

Reply:
This is something to evaluate, and probably have a rescue condition: if any of the stages fail (create the initial linux/amd image > create the rest of linux | create windows2019 | create windows2022), should we remove the image from the registry? IIRC it was super hard in the past to remove an image from a registry using CLIs/APIs, not sure if nowadays it's possible